python/lsst/ts/standardscripts/maintel/take_triplet_comcam.py

```diff
@@ -0,0 +1,248 @@
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+        # Configure tcs and camera
+        await self.configure_tcs()
+        await self.configure_camera()
+
+        # Set filter
+        self.filter = config.filter
+
+        # Set exposure time
+        self.exposure_time = config.exposure_time
+
+        # Set intra/extra focal offsets
+        self.dz = config.dz
+
+        # Set maximum number of iterations
+        self.n_triplets = config.n_triplets
+
+        # Set program and reason
+        self.reason = config.reason
+        self.program = config.program
+
+    def set_metadata(self, metadata: salobj.type_hints.BaseMsgType) -> None:
+        """Sets script metadata.
+
+        Parameters
+        ----------
+        metadata : `salobj.type_hints.BaseMsgType`
+            Script metadata topic. The information is set on the topic
+            directly.
+        """
+        # Estimated duration is maximum number of iterations multiplied by
+        # 3 (data triplet) multiplied by the time it takes to take an image
+        # plus estimation on reading out the images (10s)
+
+        metadata.duration = (
+            self.n_triplets
+            * 3
+            * (
+                self.exposure_time
+                + self.camera.read_out_time
+                + self.camera.shutter_time
+            )
+        )
+        metadata.filter = f"{self.filter}"
+
+    async def assert_feasibility(self) -> None:
+        """Verify that the telescope and camera are in a feasible state to
+        execute the script.
+        """
+        await self.mtcs.assert_all_enabled()
+        await self.camera.assert_all_enabled()
+
+    async def configure_camera(self) -> None:
+        """Handle creating ComCam object and waiting for remote to start."""
+        if self.camera is None:
+            self.log.debug("Creating Camera.")
+
+            self.camera = ComCam(
+                self.domain, log=self.log, intended_usage=ComCamUsages.TakeImage
+            )
+            await self.camera.start_task
+        else:
+            self.log.debug("Camera already defined, skipping.")
+
+    async def configure_tcs(self) -> None:
+        """Handle creating MTCS object and waiting for remote to start."""
+        if self.mtcs is None:
+            self.log.debug("Creating MTCS.")
+            self.mtcs = MTCS(
+                domain=self.domain,
+                log=self.log,
+            )
+            await self.mtcs.start_task
+        else:
+            self.log.debug("MTCS already defined, skipping.")
+
+    async def take_triplet(
+        self,
+    ) -> None:
+        """Take triplet (intra focal, extra focal and in-focus)
+        image sequence.
+        """
+        self.log.debug("Moving to intra-focal position")
+
+        await self.mtcs.move_camera_hexapod(x=0, y=0, z=self.dz, u=0, v=0)
```

Comment:
I just realized that you are using the move command and not offset. 

This will cause the hexapod to move to a position that is +dz of the LUT value, regardless of the current position of the hexapod. Is this really what you want to do? it is also going to reset any offset in x, y, u and v.

I think you want to use `offset_camera_hexapod` instead.